src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -322,7 +322,7 @@ const buildMarkers = (bpmnElement: TActivity): ShapeBpmnMarkerKind[] => {
     markers.push(ShapeBpmnMarkerKind.LOOP);
   } else if (multiInstanceLoopCharacteristics && multiInstanceLoopCharacteristics.isSequential) {
     markers.push(ShapeBpmnMarkerKind.MULTI_INSTANCE_SEQUENTIAL);
-  } else if ((multiInstanceLoopCharacteristics && !multiInstanceLoopCharacteristics.isSequential) || multiInstanceLoopCharacteristics === '') {
+  } else if (multiInstanceLoopCharacteristics && !multiInstanceLoopCharacteristics.isSequential) {
```

Comment:
The tests for this condition is not working as it should be : `BpmnJsonParser.marker.test.ts`

When I change:
```
])(`parse bpmn as json for '%s'`, (bpmnKind: string, expectedShapeBpmnElementKind: ShapeBpmnElementKind) => {
  describe.each([
    ['standardLoopCharacteristics', ShapeBpmnMarkerKind.LOOP],
    ['multiInstanceLoopCharacteristics', ShapeBpmnMarkerKind.MULTI_INSTANCE_PARALLEL],
  ])(`parse bpmn as json for '${bpmnKind}' with '%s'`, (bpmnLoopCharacteristicsKind: string, expectedMarkerKind: ShapeBpmnMarkerKind) => {
```
by 
```
])(`parse bpmn as json for '%s'`, (bpmnKind: string, expectedShapeBpmnElementKind: ShapeBpmnElementKind) => {
  describe.each([
    ['standardLoopCharacteristics', ShapeBpmnMarkerKind.LOOP],
    ['multiInstanceLoopCharacteristics', ShapeBpmnMarkerKind.LOOP],
  ])(`parse bpmn as json for '${bpmnKind}' with '%s'`, (bpmnLoopCharacteristicsKind: string, expectedMarkerKind: ShapeBpmnMarkerKind) => {
```
there is no error. It always works.

We need to fix this test before to approve this PR.

Reply:
Finally, the tests are correctly written, see https://github.com/process-analytics/bpmn-visualization-js/pull/2359#pullrequestreview-1188538060